src/roles/vsd-predeploy/tasks/vcenter.yml

```diff
@@ -57,6 +58,7 @@
       password: "{{ vcenter.password }}"
       name: "{{ vm_name }}"
       datacenter: "{{ vcenter.datacenter }}"
+      folder: "{{ vcenter.vmfolder | default(omit) }}"
```

Comment:
Is this really necessary? The docs for the module say:

This is required only, if multiple VMs with same name is found.

As long as we have a requirement that every vm name be unique, we don't need to have this, right?